src/utilities/get-page-data.ts

```diff
@@ -0,0 +1,76 @@
+import fs from 'fs'
+import glob from 'glob'
+import matter from 'gray-matter'
+import path from 'path'
+
+const getPagesSource = (source) => {
+  if (source === 'components') {
+    return path.join(
+      'node_modules',
+      '@atom-learning',
+      'components',
+      'dist',
+      'docs'
+    )
+  }
+
+  if (source === 'theme') {
+    return path.join('node_modules', '@atom-learning', 'theme', 'dist')
+  }
+
+  if (source === 'overview') {
+    return 'content'
+  }
+  return null
+}
+
+export const getPagesSlugs = async (sources: string[]) => {
+  const pageSlugs = await Promise.all(
+    sources.map((source) =>
+      glob.sync(path.join(getPagesSource(source), '**/*.{md,mdx}'))
+    )
+  )
+  const slugsBySource = sources.reduce(
+    (obj, source, i) => ({ ...obj, [source]: pageSlugs[i] }),
+    {}
+  )
+
+  return slugsBySource
+}
+
+const getMarkdownFile = (basePath, name) => {
+  const fileAsMdx = path.join(basePath, `${name}.mdx`)
```

Comment:
💬  This is a tiny one, but could we rename `fileAsMdx` and `fileAsMd` to make it clear that they're paths rather than files? I did a bit of a double take because at first glance it looked like we had `.md` and `.mdx` files with the same names.

Reply:
Yeah agreed, was trying to find a name that made sense to me and `AsFileType` made sense, `filePathAsMdx`?

Reply:
Yeah that certainly works for me!